common-lib/src/main/scala/com/gu/mediaservice/lib/aws/ThrallMessageSender.scala

```diff
@@ -40,7 +40,8 @@ case class UpdateMessage(
   crops: Option[Seq[Crop]] = None,
   mediaLease: Option[MediaLease] = None,
   leases: Option[Seq[MediaLease]] = None,
-  syndicationRights: Option[SyndicationRights] = None
+  syndicationRights: Option[SyndicationRights] = None,
+  s3Path: Option[String] = None
 ) {
```

Comment:
s3Path could be either `Option[String]` or case class for bucket and key it thrall s3 client needs that, i prefer not to have that bucket in config as it is better to decouple that